functors_task_1/student/include/impl.h

```diff
@@ -40,9 +54,28 @@ struct MixingPolicy
      * Whiskey + SevenUp = SevenPlusSeven;
      * Others + Others = Oops;
      */
-    static Cocktail mix(/**???*/)
+    static Cocktail mix(AlcoholDrink const & alcohol, NonAlcoholDrink const & nonAlcohol)
```

Comment:
Please check https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#f16-for-in-parameters-pass-cheaply-copied-types-by-value-and-others-by-reference-to-const